ReactiveCocoa/Shared/NSLayoutConstraint.swift

```diff
@@ -0,0 +1,16 @@
+import ReactiveSwift
+
+#if os(macOS)
+import AppKit
+#else
+import UIKit
+#endif
+
+extension Reactive where Base: NSLayoutConstraint {
+
+	/// Sets the constant.
+	public var constant: BindingTarget<CGFloat> {
+		return makeBindingTarget(action: { $0.constant = $1 })
```

Comment:
Can you use trailing closure syntax here? `return makeBindingTarget { $0.constant = $1 }`